### PR TITLE
[835] Add data requirements page

### PR DESCRIPTION
**Pull request by JonF1**
### Context
https://trello.com/c/Fn0yqXbr/835-s-data-requirements-page

### Changes proposed in this pull request
* Added data requirements page that can be reached from home page in future PR
* Added controller test
### Guidance to review
Navigate to /data-requirements to see markup rendered

screencapture-localhost-5000-data-requirements-2021-01-12-17_39_13

### CODE REVIEW

**Comment by JonF1** [comment]
[[S] Data Requirements Page](https://trello.com/c/Fn0yqXbr/835-s-data-requirements-page)

File: app/views/pages/data_requirements.html.erb [hunk 1]
```diff
@@ -0,0 +1,88 @@
+<%= render PageTitle::View.new(title: "pages.data_requirements") %>
+
+<%= content_for(:breadcrumbs) do %>
+  <%= render GovukComponent::BackLink.new(
+    text: 'Back',
+    href: "/home"
```
Comment (defong) [inline app/views/pages/data_requirements.html.erb:6]: :thinking: 


image

File: app/views/pages/data_requirements.html.erb [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    </ul>
+
+    <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
+
+    <div class="govuk-heading-m">Allocations information</div>
+
+    <p class="govuk-body">If the course has allocated places, you’ll need to know whether the trainee is using an allocated place.</p>
+
+  </div>
+</div>
```
Comment (defong) [inline app/views/pages/data_requirements.html.erb:88]: line ending

**Review by defong — COMMENTED** [review]
https://rtt-review-pr-386.herokuapp.com/data-requirements

Minor bits otherwise it looks fine to me

File: app/views/pages/data_requirements.html.erb (hunk 1 above)
Comment (JonF1) [reply app/views/pages/data_requirements.html.erb:6]: think @avinhurry is doing the home page at the moment, it's just a placeholder till then

File: app/views/pages/data_requirements.html.erb (hunk 2 above)
Comment (JonF1) [reply app/views/pages/data_requirements.html.erb:87]: fixed

File: app/views/pages/data_requirements.html.erb [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+  ) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds-from-desktop">
+    <span class="govuk-heading-l govuk-!-margin-bottom-9">Check what data you need</span>
+
+    <p class="govuk-body">You’ll need the following information about a trainee to complete their record.</p>
+
+    <div class="govuk-heading-m">Personal Details</div>
```
Comment (AbigailMcP) [inline app/views/pages/data_requirements.html.erb:16]: Lowercase 'details'

File: app/views/pages/data_requirements.html.erb [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+      <li>Middle names</li>
+      <li>Last or family name</li>
+      <li>Date of birth</li>
+      <li>Gender</li>
+      <li>Nationality</li>
+    </ul>
+
+    <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
+
+    <div class="govuk-heading-m">Contact Details</div>
```
Comment (AbigailMcP) [inline app/views/pages/data_requirements.html.erb:29]: As above, lowercase 'details'

File: app/views/pages/data_requirements.html.erb [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+    </ul>
+
+    <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
+
+    <div class="govuk-heading-m">Degrees</div>
+
+    <div class="govuk-heading-s">If it's a UK degree</div>
+
+    <ul class="govuk-list">
+      <li>Type</li>
```
Comment (AbigailMcP) [inline app/views/pages/data_requirements.html.erb:54]: Should read: Type (for example, BA, BSc or other)

**Review by AbigailMcP — COMMENTED** [review]
Did a quick side-by-side with the prototype and spotted a couple of things, other than that it's all good!

File: app/views/pages/data_requirements.html.erb (hunk 5 above)
Comment (JonF1) [reply app/views/pages/data_requirements.html.erb]: thanks, fixed

**Review by AbigailMcP — APPROVED** [review]

**Pull request merged**